src/code/z_player_lib.c

```diff
@@ -1339,22 +1339,18 @@ void func_80090D20(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3s*
         } else if (limbIndex == PLAYER_LIMB_HEAD) {
             Matrix_MultVec3f(&D_801260D4, &this->actor.focus.pos);
         } else {
-            Vec3f* vec = &D_801261E0[(0, gSaveContext.linkAge)];
+            Vec3f* vec = &D_801261E0[((void)0, gSaveContext.linkAge)];
 
             Actor_SetFeetPos(&this->actor, limbIndex, PLAYER_LIMB_L_FOOT, vec, PLAYER_LIMB_R_FOOT, vec);
         }
     }
 }
 
 u32 func_80091738(GlobalContext* globalCtx, u8* segment, SkelAnime* skelAnime) {
-    s16 linkObjectId;
-    u32 size;
-    void* ptr;
+    s16 linkObjectId = gLinkObjectIds[(void)0, gSaveContext.linkAge];
+    u32 size = gObjectTable[OBJECT_GAMEPLAY_KEEP].vromEnd - gObjectTable[OBJECT_GAMEPLAY_KEEP].vromStart;
+    void* ptr = segment + 0x3800;
```

Comment:
It's not a huge deal, but I personally disagree with combining the declarations in cases like these. Here and in some other places, it makes more sense to leave them separate imo.

Reply:
Could you explain a little more why you think so? I'm currently under the impression we should be doing this wherever we can, not limited to constants.

Reply:
Further down in the function, `size` and `ptr` are assigned values before the `DmaMgr_SendRequest1` call, so it's more consistent to set their values before the function call in both instances.

And in general, initializing the variable in the declaration doesn't necessarily make sense if the variable is written to later on in the function. (this is why I didn't combine them in functions like `Item_Give`, for instance)